src/components/notification/Notification.tsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect } from 'react';
+
+interface Props {
+  setError: (errVal: string | boolean) => void;
+  errorText: string | true;
```

Comment:
Replace 'string | true' with 'string' since 'true' is not a valid value for errorText.